app/core/components/TextEditor/index.tsx

```diff
@@ -1,13 +1,34 @@
-import { TextareaAutosize } from "@mui/material";
 import { useControlField } from "remix-validated-form";
+import { Suspense, lazy, useEffect, useState } from "react";
+import type { ReactNode } from "react";
+
+let LazyMDEditor = lazy(() => import("@uiw/react-md-editor"));
 
 type TextEditorProps = {
   name: string;
   placeholder?: string;
 };
 
+export function ClientOnly({ children }: { children: ReactNode }) {
+  let [mounted, setMounted] = useState(false);
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+  return mounted ? <>{children}</> : null;
```

Comment:
We could use the light theme, in order to have the same style.
Like this` return mounted ? <div data-color-mode="light">{children}</div> : null;`

Reply:
Ah this is because you have dark mode enabled in your OSS. But if you switch to light theme it should look ok. By the way #98 enables dark mode using MUI styles, so I think we should leave this functionality as it is.